lib/azure/armrest/network/subnet_service.rb

```diff
@@ -0,0 +1,72 @@
+# Azure namespace
+module Azure
+  # Armrest namespace
+  module Armrest
+    # Base class for managing subnets
+    class SubnetService < VirtualNetworkService
```

Comment:
I guess I should, but the class names are already a bit long. I guess I was just trying to save my fingers a bit...